public/app/plugins/panel/datagrid/utils.ts

```diff
@@ -79,6 +72,17 @@ interface CellRange {
   height: number;
 }
 
+export const updateSnapshot = async (
```

Comment:
Don't use variable assigned a arrow functions unless your have two (usually only requires in classes with callbacks to bind this), instead use a named function (easier to read and crates better stack traces )

Reply:
Thanks for that @torkelo. I pushed a modification